spec/datadog/opentelemetry_spec.rb

```diff
@@ -339,18 +339,21 @@
       describe '#inject' do
         subject(:inject) { ::OpenTelemetry.propagation.inject(carrier) }
         let(:carrier) { {} }
-
+        let(:headers) do
+          {
+            'x-datadog-parent-id' => Datadog::Tracing.active_span.id.to_s,
+            'x-datadog-sampling-priority' => '1',
+            'x-datadog-tags' => '_dd.p.dm=-0,_dd.p.tid=' +
+              Datadog::Tracing::Utils::TraceId.to_high_order(Datadog::Tracing.active_trace.id).to_s(16),
+            'x-datadog-trace-id' => Datadog::Tracing::Utils::TraceId.to_low_order(Datadog::Tracing.active_trace.id).to_s,
```

Comment:
@marcotc I added a [helper for this](https://github.com/DataDog/dd-trace-rb/pull/3266/files#diff-e7ca68de5426a8ab8980827a9671f1b74ffe6c5b6d3a91eaf0201be601e407abR141) but it's scoped to `Tracing::Contrib` so I can't / shouldn't use it outside of that.